src/components/GlobalFooter/GlobalFooter.tsx

```diff
@@ -1,27 +1,24 @@
-import React, { HTMLAttributes } from 'react';
+import React from 'react';
 import { cnb } from 'cnbuilder';
 import { StanfordLogo } from '../StanfordLogo';
 import { SrOnlyText } from '../Typography';
 import { Container } from '../Container';
 import { FlexBox } from '../FlexBox';
 import * as styles from './GlobalFooter.styles';
 
-type GlobalFooterProps = Omit<HTMLAttributes<HTMLDivElement>, 'className'> & {
+type GlobalFooterProps = {
   className?: string;
 };
 
-const GlobalFooter = ({ className, ...rest }: GlobalFooterProps) => (
-  <Container
-    {...rest}
-    className={cnb(styles.root, className)}
-    width="site"
-  >
+const GlobalFooter = ({ className }: GlobalFooterProps) => (
+  <Container className={cnb(styles.root, className)}>
```

Comment:
I don't need to pass anything into the Global Footer component other than classNames so removing the {...rest} which get rid of the slicecontext warning also